packages/app/src/files-manager/components/files-summary-item/files-summary-item.stories.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { storiesOf } from "@storybook/react"
+import * as React from "react"
+import { Router } from "react-router"
+import FilesSummaryItem from "App/files-manager/components/files-summary-item/files-summary-item.component"
+import history from "Renderer/routes/history"
+import { Type } from "Renderer/components/core/icon/icon.config"
+
+const fakeData = {
```

Comment:
🟡
Interface missing. The interface is important during refactoring than lint can easily catch errors.

Reply:
Added in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)